setup.py

```diff
@@ -41,7 +40,7 @@ class build_py_with_git_version(build_py):
     def build_module(self, module, module_file, package):
         orig_outfile, _ = build_py.build_module(self, module, module_file,
                                                 package)
-        if module == 'eucameta' and package == 'sos.plugins':
+        if module == 'VERSION':
```

Comment:
`build_module` is only called on modules.  Since VERSION.py is not a module this condition is never true.  This entire class can thus be removed.